src/Navbar.js

```diff
@@ -0,0 +1,32 @@
+import React from "react"
```

Comment:
Hey Devan, in general for our website we use typescript for the frontend, it follows similar syntax to javascript but it includes things like static typing and better intellisense. If you don't mind, try converting your Navbar.js and Team.js into typescript files. For example, the props of your CustomLink component can be wrapped inside an interface which if you know C is somewhat similar to a struct where you just define the data and its types.